src/test/java/no/entur/uttu/export/blob/S3BlobStoreRepositoryTest.java

```diff
@@ -210,7 +218,29 @@ private void assertBlobExists(
       Assert.fail(bucket + " / " + key + " does not exist");
     }
     if (expectedMetadata != null) {
-      Assert.assertEquals(response.metadata(), expectedMetadata);
+      Assert.assertEquals(expectedMetadata, mimeDecodeValues(response.metadata()));
     }
   }
+
+  private static Map<String, String> mimeDecodeValues(Map<String, String> metadata) {
```

Comment:
There are no metadata accessing methods in `BlobStoreRepository` so the necessary decoding is now done in test only. If the base interface ever gains such abilities, this method should of course be promoted there to make this work transparently.